devel/tag-latest

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Assign the given additional tags to the nextstrain/base:latest and
+# nextstrain/base-builder:latest images.
+# Errors if a tagged image has already been pushed.
+#
+set -euo pipefail
+
+if [[ $# -ne 1 ]]; then
+    echo "Please provide a tag." >&2
+    exit 1
+fi
+
+tag=$1
+
+BASE_IMAGE="nextstrain/base"
+BASE_BUILDER_IMAGE="nextstrain/base-builder"
+
+docker tag $BASE_BUILDER_IMAGE:{$tag,latest}
+docker tag $BASE_IMAGE:{$tag,latest}
```

Comment:
[This currently doesn't work](https://github.com/nextstrain/docker-base/runs/7735250010?check_suite_focus=true#step:11:14):

> Error response from daemon: No such image: nextstrain/base-builder:branch-victorlin-auto-push-named-tag

Seems like the tag is ephemeral in [build-tag-push](https://github.com/nextstrain/docker-base/blob/1e27068d571a5138967c21a338c478bff0dbea44/devel/build-tag-push).

Reply:
This was due to the switch from `--load` to `--push`. This is no longer an issue after switching back to `--load`.